laptop_install_test

```diff
@@ -119,3 +120,44 @@ do
     log_install_success "$gui_app_name"
   fi
 done
```

Comment:
Nice.. Worth also adding a check for existence of `pip`?

Reply:
Good shout! Totally slipped my mind!